### PR TITLE
Initial sky atmosphere documentation

### PR DESCRIPTION
**Pull request by AMZN-alexpete**
## Change summary

-add initial documentation and examples about how to use the Sky atmosphere component

Levels for these examples are uploaded here https://github.com/AMZN-alexpete/o3de-demo/releases/download/0.0.1/SkyExampleLevels.zip

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by galibzon — APPROVED** [review]

**Comment by galibzon** [comment]
Nice work! Love the inter-planetary picture.

File: content/docs/user-guide/components/reference/atom/sky-atmosphere.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+The **Sky Atmosphere** component provides physically-based sky and atmosphere rendering capabilities.
+
+The sky is rendered by sampling points on rays that start at the camera and intersect with the atmosphere.  Because sampling many rays every frame is an expensive operation, luminence and volume look up tables are used to perform fewer calculations.  
+
+![Sky atmosphere in desert mountain scene](/images/user-guide/components/reference/atom/sky-atmosphere/desert-day.jpg)
+
+## Provider
+
+[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
```
Comment (tkothadev) [inline content/docs/user-guide/components/reference/atom/sky-atmosphere.md:16]: although this is not the developer guide, perhaps an extra blurb describing how atom provides for the sky atmosphere could make it easier to understand how to work with it?

File: content/docs/user-guide/components/reference/atom/sky-atmosphere.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+|-|-|-|-|
+| **Origin** | The origin to use for the atmosphere.  When `Ground at world origin` (default) is selected, the bottom of the atmosphere - the ground - will be at the world origin, or `0,0,0`.  If `Ground at local origin` is used, then the bottom of the atmosphere will be at the world position of the Transform Component for the entity the Sky Atmosphere component is on.  Using `Ground at local origin` gives you an easy way to control the position of the bottom of the atmosphere.  One practical use for `Ground at local origin` is to move the ground position down so you have a more gradual gradient transition from surface to sky.  Try using `Ground at local origin` and moving the entity transform down by setting the `Z` value to `-100` to get a smoother transition. The `Planet center at local origin` is makes the center of the planet for the atmosphere at the world position of the Transform component for the entity the Sky Atmosphere component is on.  The `Planet center at local origin` is useful when you want to have atmosphere for planets. | `Ground at world origin`,`Ground at local origin`, `Planet center at local origin`| `Ground at world origin` |
```
Comment (tkothadev) [inline content/docs/user-guide/components/reference/atom/sky-atmosphere.md:24]:
```suggestion
| **Origin** | The origin to use for the atmosphere.  When `Ground at world origin` (default) is selected, the bottom of the atmosphere - the ground - will be at the world origin, or `0,0,0`.  If `Ground at local origin` is used, then the bottom of the atmosphere will be at the world position of the Transform Component for the entity the Sky Atmosphere component is on.  Using `Ground at local origin` gives you an easy way to control the position of the bottom of the atmosphere.  One practical use for `Ground at local origin` is to move the ground position down so you have a more gradual gradient transition from surface to sky.  Try using `Ground at local origin` and moving the entity transform down by setting the `Z` value to `-100` to get a smoother transition. The `Planet center at local origin` makes the center of the planet for the atmosphere at the world position of the Transform component for the entity the Sky Atmosphere component is on.  The `Planet center at local origin` is useful when you want to have atmosphere for planets. | `Ground at world origin`,`Ground at local origin`, `Planet center at local origin`| `Ground at world origin` |
```

Comment (tkothadev) [reply content/docs/user-guide/components/reference/atom/sky-atmosphere.md:24]: there was a small typo on the sentence `The `Planet center at local origin` is makes the center of...`

**Comment by tkothadev** [comment]
minor comments, but looks good to me

**Review by tkothadev — APPROVED** [review]

**Pull request merged**